draft-ietf-gnap-core-protocol.md

```diff
@@ -1728,9 +1734,9 @@ Additional interaction modes are defined in [a registry TBD](#IANA).
 
 \[\[ [See issue #61](https://github.com/ietf-wg-gnap/gnap-core-protocol/issues/61) \]\]
 
-## Declaring RC Capabilities {#request-capabilities}
+## Declaring Client Instance Capabilities {#request-capabilities}
```

Comment:
Wouldn't this be more accurate to leave out "instance" in this section? Capabilities are properties of the software.